### ISSUE TITLE
Docker support

### ISSUE THREAD

**Issue by IamADav** [Azure/Azurite#94]
Would it be possible to have a new docker image. We were used to download them from arafato/azurite repository but it seems that there is no image repository anymore.

Thanks for your work.

**Comment by cswendrowski** [comment]
I am temporarily building and hosting the Docker images of this repository until deployments start coming out of this repo again: https://hub.docker.com/r/ironmoosedevelopment/azurite/

**Comment by XiaoningLiu** [comment]
We are moving Azurite Docker image to Microsoft official accounts, stay tuned..

**Comment by ahoka** [comment]
Any update?

**Comment by XiaoningLiu** [comment]
Should get some updates in next week.

**Comment by XiaoningLiu** [comment]
We currently published 2.6.7 into microsoft MCR docker. Will update docker image page sonn

docker pull mcr.microsoft.com/azurestorage-azurite

docker pull mcr.microsoft.com/azurestorage-azurite:2.6.7

**Comment by thisisthekap** [comment]
@XiaoningLiu When will the new docker image be available in the repository?

PS: Are you also going to update README.md to reflect the changes made to the azurite docker setup?

**Comment by XiaoningLiu** [comment]
Yes. Still need some time for the docker hub page/repository publish. Azurite 2.6.7 and 2.7.0 should have the same feature set. You can use the 2.6.7.

**Comment by thisisthekap** [comment]
@XiaoningLiu Any update on the docker support?

**Comment by jakirkham** [comment]
Just checking in, is there a rough time estimate.

**Comment by jakirkham** [comment]
Just checking in, is there a rough time estimate?

**Comment by XiaoningLiu** [comment]
Sorry, we didn't have a timeline right now, but we will try our best to have a update. You can relay on the 2.6.7 docker image. Azurite is just planned as our focus on next semester.
